### DISCUSSION TITLE
Calculate distance between periodic images

### DISCUSSION THREAD

**Discussion by H-EKE** [MDAnalysis/mdanalysis#4764] [Q&A] [answered]
Hi,

I ran a MD simulations where I have a huge conformational change. I am worried that my inital box guess was too short, and now the periodic images are interacting between them. Is there a way to check if the protein is interacting whith its periodic image in MDAnalysis?

I thought about something like this, but Im not sure what to put as second selection (protein2)

```

import MDAnalysis as mda
from MDAnalysis.analysis import distances

u = mda.Universe('structure.psf', 'output.xtc')

protein = u.select_atoms('name CA') 
protein2 = u.select_atoms('name CA',periodic=True) 
distance_array = distances.distances_array(protein.positions, protein2.positions, box=u.dimensions)
```

**Comment by orbeckst** [comment, marked as answer]
The `periodic=True` does not do anything for atomgroups.

I don't think that there's a built-in way to do exactly what you want in MDAnalysis. However, we do have the [MDAnalysis.lib.distances.augment_coordinates()](https://docs.mdanalysis.org/stable/documentation_pages/lib/distances.html#MDAnalysis.lib.distances.augment_coordinates) function. You might be able to generate copies and then compute distances to the copies. Not sure how to do it in detail but this may at least be a starting point.

**Comment by H-EKE** [comment]
Dear @orbeckst 

I will try your suggestion. Thanks a lot !
